src/main/java/edu/wpi/first/smartdashboard/gui/elements/CameraServerViewer.java

```diff
@@ -23,7 +23,12 @@
       return Stream.empty();
     }
 
-    return Arrays.stream(cameraTable.getStringArray(STREAMS_KEY, new String[0]));
+    return Arrays.stream(cameraTable.getStringArray(STREAMS_KEY, new String[0])).map(s -> {
+      if (NetworkTable.connections().length > 0) {
+        return s.replaceFirst("roboRIO-\\d+-FRC.*(?=:)", NetworkTable.connections()[0].remote_ip);
```

Comment:
CameraServer already seems to do this, if you use the second stream in the array. That seems safer them making your own substitution.